app/controllers/addressLookupFrontend/RampOffController.scala

```diff
@@ -49,7 +50,7 @@ class RampOffController @Inject()(controllerActions: ControllerActions,
         updatedUserAnswers = addressLookupService.addAddressUserAnswers(PackingDetails, alfResponse.address, request.userAnswers, sdilId, alfId)
         _ <- sessionRepository.set(updatedUserAnswers)
       } yield {
-        Redirect(controllers.routes.IndexController.onPageLoad)
+        Redirect(controllers.changeActivity.routes.PackagingSiteDetailsController.onPageLoad(NormalMode))
```

Comment:
This should be a conditional redirect url as it is a function shared across mini journeys so it should be

`val redirectUrl = updatedUserAnswers.journeyType match {
case Some(SelectChange.ChangeActivity) => controllers.changeActivity.routes.PackagingSiteDetailsController.onPageLoad(NormalMode)
case _ => controllers.routes.IndexController.onPageLoad
}

Redirect(redirectUrl)
`

Reply:
updated